stored/backends/gs.py

```diff
@@ -46,6 +46,9 @@ def sync_to(self, output_path):
             for path in new_paths:
                 GoogleStorage(os.path.join(self.url, path)).sync_to(os.path.join(output_path, path))
         else:
+            output_dir = os.path.dirname(output_path)
+            if not os.path.exists(output_dir):
+                os.makedirs(output_dir)
```

Comment:
It's so weird that there's no utility method to just do this automagically

Reply:
Python 3 does using `exist_ok=True` but we want support for Python 2 too